db.go

```diff
@@ -59,6 +60,7 @@ type Executor interface {
 	ExecContext(ctx context.Context, query interface{}, args ...interface{}) (sql.Result, error)
 	GetContext(ctx context.Context, dest interface{}, keys ...interface{}) error
 	InsertContext(ctx context.Context, list ...interface{}) error
+	InsertIgnoreContext(ctx context.Context, list ...interface{}) error
```

Comment:
N.B. breaks existing `Executor` implementations, but there doesn't seems to be an alternative.